### PR TITLE
adding support for automatic loading by default

### PR DESCRIPTION
**Pull request by rodriigovieira**

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -169,15 +178,27 @@ export function persistTree(structure : any, store : any, debug: boolean) {
...
-    if(debug) {
-        _map(structure, async (object : any, name : any) => {
+    if (debug) {
+        _map(structure, async (object: any, name: any) => {
             console.log(object.action || buildAction(name));
         });
     }
 }
 
+function _loadAutomaticReducers(store: any) {
```
Comment (lukebrandonfarrell) [inline lib/index.ts:191]: Looking good! 

Could we remove the `_` from this function, and just have it be called `loadAutomaticReducers`?

Also, would you be able to add comments to it, etc:



```
/**
* What does this function do?
*/
function loadAutomaticReducers(store: any) {

```

File: tsconfig.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
     "declaration": true,
     "outDir": "./lib",
     "strict": true,
-    "lib": ["es6", "es2017.object"],
+    "lib": ["es6", "es2017.object", "DOM"],
```
Comment (lukebrandonfarrell) [inline tsconfig.json:9]: What is the requirement for "DOM" addition?

**Review by lukebrandonfarrell — CHANGES_REQUESTED** [review]
Looks solid @rodriigovieira !

File: tsconfig.json (hunk 2 above)
Comment (rodriigovieira) [reply tsconfig.json:9]: It was required to run `tsc`, because we're using `console.log`.

**Pull request merged**